model/project_ref.go

```diff
@@ -2390,7 +2392,8 @@ func (p *ProjectRef) GetGithubProjectConflicts() (GithubProjectConflicts, error)
 	}
 
 	for _, conflictingRef := range matchingProjects {
-		if conflictingRef.Id == p.Id {
+		// If this is the same project ref or this project ref will inherit from it, it is not conflicting.
+		if conflictingRef.Id == p.Id || conflictingRef.Id == p.RepoRefId {
```

Comment:
I believe we want the opposite of the 2nd boolean statement here, i.e. `if conflictingRef.Id == p.Id || conflictingRef.RepoRefId == p.Id`, because `RepoRefId` is the ID of the repo that a project ref inherits its values from.

That way, if we are modifying some settings on the repo, we skip checking conflicts for all `matchingProjects` that are project refs attached to the repo, since checking for conflicts isn't relevant/helpful.